src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -341,25 +349,5 @@ private static void Add(IList<string> cmd, string prefix, string value)
                 cmd.Add(value);
             }
         }
-
-        internal class Capture
```

Comment:
`FileHelper` has two internal classes, `FileConfig` and `Capture`. I'm not sure what the point of `Capture` is. However, I've separate out the Capture-related code and tests into separate files to make it easier to see what is going on.

Reply:
I believe Capture is now the class that contains the post-processing logic mentioned in https://github.com/SonarSource/sonarlint-visualstudio/issues/1524